tests/unit/AGMDate.spec.ts

```diff
@@ -5,19 +5,10 @@ import { mount, Wrapper } from '@vue/test-utils'
 import { getVuexStore } from '@/store'
 import AgmDate from '@/components/AnnualReport/AGMDate.vue'
 
-// NB: test util async issue
-// in some cases, the elements are not updated during the test
-// the work-around is to first initialize the property we are changing
-// suppress update watchers warnings
+// suppress "Avoid mutating a prop directly" warnings
```

Comment:
More concise message specific to this file.
(And ditto for all other similar changes below.)